crates/lox_core/src/time/leap_seconds/gen.rs

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023. Helge Eichhorn and the LOX contributors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// From here: https://github.com/JuliaTime/LeapSeconds.jl/blob/master/gen/leap_seconds.jl
+
+pub const LS_EPOCHS: [u64; 28] = [
```

Comment:
We should probably model this as seconds from J2000 instead.

Reply:
So ((x * MJD_EPOCH)  - 2451545.0) * SECONDS_PER_DAY?

Reply:
https://github.com/lox-space/lox/issues/31